pom.xml

```diff
@@ -189,6 +193,33 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.1.11</version>
+          <configuration>
+            <reportsDirectory>local/pit-reports</reportsDirectory>
+            <timestampedReports>false</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <targetClasses>
+              <param>${pitest.classes}</param>
```

Comment:
Does the pitest-maven plugin really need to know "org.optaplanner*"?

Can't it just grab all the classes from the current module? Suppose the module is optaplanner-examples, does pitests need to change optaplanner-core classes then?